### DISCUSSION TITLE
screen BTT TFT4_3

### DISCUSSION THREAD

**Discussion by 2ed888** [teeminus/NoTouchScreenFirmware#55] [Show and tell]
guys, today I checked with the BTT TFT4_3V3_0 screen - everything works
checked with a board SKR V1.4 turbo

**Comment by teeminus** [comment]
Thanks for the feedback 👍

Which mainboard firmware have you tested with? If you used Klipper and the config for the mainboard/display combo is not listed in the wiki, would you please share the config with us?

**Reply by 2ed888** [reply]
the types of board and ekranoga indicated above ... and the clipper firmware of the latest build ... updated it before checking

**Reply by teeminus** [reply]
Could you please share your Klipper display config for the wiki?

**Reply by 2ed888** [reply]
Yes, please!

[display]
lcd_type: st7920
cs_pin: EXP1_4
sclk_pin: EXP1_5
sid_pin: EXP1_3
encoder_pins: ^EXP2_3, ^EXP2_5
click_pin: ^!EXP1_2
#kill_pin: ^!EXP2_8

[output_pin beeper]
pin: EXP1_1
pwm: True
value: 0
shutdown_value: 0
cycle_time: 0.0024



[board_pins]
aliases:
    # EXP1 header
    EXP1_1=P1.30, EXP1_3=P1.18, EXP1_5=P1.20, EXP1_7=P1.22, EXP1_9=<GND>,
    EXP1_2=P0.28, EXP1_4=P1.19, EXP1_6=P1.21, EXP1_8=P1.23, EXP1_10=<5V>,
    # EXP2 header
    EXP2_1=P0.17, EXP2_3=P3.26, EXP2_5=P3.25, EXP2_7=P1.31, EXP2_9=<GND>,
    EXP2_2=P0.15, EXP2_4=P0.16, EXP2_6=P0.18, EXP2_8=<RST>, EXP2_10=<NC>
    # Pins EXP2_1, EXP2_6, EXP2_2 are also MISO, MOSI, SCK of bus "ssp0"

**Comment by 2ed888** [comment]
and ... guys, English is not native, please do not kick your feet ... thanks!

**Comment by 2ed888** [comment]
guys, clarification:
1.with layout
[display]
lcd_type: st7920
cs_pin: EXP1_4
sclk_pin: EXP1_5
sid_pin: EXP1_3
Encoder_pins: ^ EXP2_5, ^ EXP2_3
click_pin: ^! EXP1_2
works very unstable, it starts up one time out of seven, ten.
2.checked with the layout:
[display]
lcd_type: emulated_st7920
en_pin: PD10
spi_software_sclk_pin: EXP1_5
spi_software_mosi_pin: EXP1_3
spi_software_miso_pin: EXP1_6
encoder_pins: ^ EXP2_5, ^ EXP2_3
click_pin: ^! EXP1_2
works steadily, but the most important thing: do not press any buttons during operation - reset the screen! otherwise the screen will stop working and you will need to restart the raspberry
